packages/data/src/cluster/ClusterIndex.ts

```diff
@@ -197,13 +206,14 @@ export function buildIndex<F>(clusterLevels: ClusterLevels): ClusterIndex<F> {
               dest: destCluster,
               count: flowCountsMapReduce.map(flow),
               aggregate: true,
+              values: [flowCountsMapReduce.map(flow)],
             };
             result.push(aggregateFlow);
             aggFlowsByKey.set(key, aggregateFlow);
           } else {
+            aggregateFlow.values.push(flowCountsMapReduce.map(flow));
```

Comment:
Probably `aggregateFlow.values.push(flowCountsMapReduce.map(flow))` is not doing what was intended:
<img src=https://user-images.githubusercontent.com/351828/213997071-5ebf26fa-51e4-4642-85bd-f3c56fd89a8b.png width=300>

Reply:
I made the required changes to apply the plugged aggregation function on the node clustering and volume function.